src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+@use '../utiles/variables' as *;
+
+.header_bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+}
+
+.navigation_bar {
```

Comment:
in header file there should only be styles for header block. You need to create a separate file for navigation and change the logo to belong to header